<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDz</class>
 <widget class="QWidget" name="CheckDz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>741</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1099</width>
      <height>739</height>
     </rect>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
